packages/extension/src/companion/CompanionContextMenu.tsx

```diff
@@ -78,9 +75,21 @@ export default function CompanionContextMenu({
             <CommentIcon size="medium" className="mr-2" /> View discussion
           </a>
         </Item>
-        <Item onClick={() => onShareOrCopyLink()}>
-          <ShareIcon size="medium" className="mr-2" /> Share article
-        </Item>
+        {additionalInteractionButtonFeature ===
+        AdditionalInteractionButtons.Bookmark ? (
+          <Item onClick={onShare}>
+            <ShareIcon size="medium" className="mr-2" /> Share article via...
+          </Item>
+        ) : (
+          <Item onClick={onBookmark}>
+            <BookmarkIcon
+              size="medium"
+              className="mr-2"
+              filled={postData?.bookmarked}
+            />{' '}
```

Comment:
I think the extra space shouldn't be necessary since we applied the margin to the right.
```suggestion
            />
```